src/app/components/layout/Tooltip.scss

```diff
@@ -0,0 +1,45 @@
+@import '../../../scss/fonts';
+
+.tooltip {
+    position: relative;
+    display: flex;
+
+    .wrap {
```

Comment:
Rename `.wrap` class so it is not generic and lets continue to use BEM naming convention. ex. `.tooltip__content` or `.tooltip__title`  for the element of the tooltip block

Reply:
Done.